src/env.c

```diff
@@ -408,7 +403,7 @@ static unsigned char *tls_from_tls_base(uintptr_t tls_base)
 
 static unsigned char *tls_from_tls_region(unsigned char *tls_region)
 {
-    return tls_from_tls_base(tls_base_from_tls_region(tls_region));
+    return tls_addr_from_tls_base(tls_base_from_tls_region(tls_region));
```

Comment:
Should this be paired with a change to the name of the function itself? If so, can that be done in a separate commit?

Reply:
This is a typo, and needs to be reverted.